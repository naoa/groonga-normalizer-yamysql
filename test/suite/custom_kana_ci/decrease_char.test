register normalizers/yamysql

normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml "ヴァイオリンの演奏" WITH_TYPES|WITH_CHECKS
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml "ﾊﾞｲｵﾘﾝの演奏" WITH_TYPES|WITH_CHECKS
