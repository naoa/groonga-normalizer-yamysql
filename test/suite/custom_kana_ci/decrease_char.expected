register normalizers/yamysql
[[0,0.0,0.0],true]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml "ヴァイオリンの演奏" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "バイオリン 演奏",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null",
      "kanji",
      "kanji"
    ],
    "checks": [
      6,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
normalize NormalizerYaMySQLKanaCISymbolPartofspeechHtml "ﾊﾞｲｵﾘﾝの演奏" WITH_TYPES|WITH_CHECKS
[
  [
    0,
    0.0,
    0.0
  ],
  {
    "normalized": "バイオリン 演奏",
    "types": [
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "katakana",
      "null",
      "kanji",
      "kanji"
    ],
    "checks": [
      6,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      0,
      0,
      3,
      3,
      0,
      0,
      3,
      0,
      0
    ]
  }
]
